conf/application.conf

```diff
@@ -38,7 +38,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
 play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.MicroserviceFilters"
 
 # Json error handler
-play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler"
+play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandler"
```

Comment:
realised we'd missed this in the upgrades - `uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler` is now deprecated